apps/web/src/components/patient-documents/PatientDocumentUpload.tsx

```
@@ -0,0 +1,687 @@
+              <Progress
+                value={(totalDocuments / maxFiles) * 100}
+                className='h-2'
+                aria-label={`${totalDocuments} de ${maxFiles} documentos enviados`}
+              />
+            </div>
+          </div>
+        </div>
+
+        {/* Upload Queue */}
+        {uploadQueue.size > 0 && (
+          <div className='space-y-3' role='region' aria-label='Progresso de upload'>
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)